src/kyber512/indcpa.c

```diff
@@ -28,10 +28,11 @@ static void matacc(poly* r, polyvec *b, unsigned char i, const unsigned char *se
   unsigned int ctr, pos, k, l;
   uint16_t val0, val1;
   int16_t c[4];
+  int j;
```

Comment:
Because this way of declaring integers inside of for loops is a feature ofC-99, which the windows compiler we recommend cannot compile correctly while also compiling the assembly statements we use.

Reply:
However, since we'll stop recommending that people compile the pinata firmware on windows (without wsl) all together in the next pinata documentation, I'm reverting this change. It is not needed.

Reply:
But this code is not meant to be compiled with MSVC.